### PR TITLE
Change format of report & hotfix

### PR DESCRIPTION
**Pull request by CEJ-R**
This PR changes the output of the weekly report and adds hotfix to skip "corrected_omicron_consensus" folder.

To test:
`conda activate /apps/bio/repos/sars-cov-2-typing/conda-env/sars-cov-2-typing`
`python weekly_report.py -o ../test_output_file.txt`

### CODE REVIEW

File: weekly_report.py [hunk 1]
```diff
@@ -514,6 +516,13 @@ def write_uploaded_samples(uplodedSamplesFile, outf):
         # add a total value of all of uploaded samples 
         data[row['sequencing_week']]['total'] = data[row['sequencing_week']]['unknown'] + data[row['sequencing_week']]['selection'] + data[row['sequencing_week']]['general']
 
+    # Read data into a dataframe, group by upload week, and sum the values
+    df = pd.DataFrame.from_dict(data).T
+    df =df.groupby(['upload_week'],as_index = False).sum()
+    # Write dataframe to outputfile
+    df.to_csv(outf, sep='\t', index=False)
+
+    """
```
Comment (sima-r) [inline weekly_report.py:525]: You can remove these commented lines

Comment (CEJ-R) [reply weekly_report.py]: Done in latest commit.

**Comment by sima-r** [comment]
@CEJ-R I just merged the PR. You can delete this report branch and I guess open another PR from dev to main? (Don't remember if we're going to keep the dev branch as the active working branch or it should be main.)

**Comment by CEJ-R** [comment]
@sima-r I made a separate PR for this into main and set you and Anders to review (again). I guess all working code should be on main. So if we agree that this PR works, then I suppose it could be merged?

**Comment by sima-r** [comment]
Sounds fine to me

**Pull request merged**